javascript/javascript1/week3/series-duration.js

```diff
@@ -0,0 +1,54 @@
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Sopranos",
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: "The Wire",
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    },
+    {
+      title: "Haunted",
+      days: 0,
+      hours: 7,
+      minutes: 30,
+    },
+    {
+      title: "The Crown",
+      days: 1,
+      hours: 18,
+      minutes: 10,
+    }
+    ];
+  let lifeSpanInMins = 80 * 365 * 24 * 60;
```

Comment:
This should be a `const`.
And be sure to use the full words:
`lifeSpanInMinutes`

Reply:
ok